var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -152,5 +152,7 @@ def configure_args(self):
         if "+squashfuse" in self.spec:
             squashfuse_prefix = "{0}".format(self.spec["squashfuse"].prefix)
             args.append("--with-libsquashfuse={0}".format(squashfuse_prefix))
+            fuse_include = self.spec["fuse"].prefix.include.fuse3
+            args.append("CFLAGS=-I{0}".format(fuse_include))
```

Comment:
Can you implement this with a `flag_handler` method like in https://spack.readthedocs.io/en/latest/packaging_guide.html#compiler-flags?

If you define a function like 

```
def flag_handler(self, name, flags):
    if name == "cflags":
        flags.append(f"-I{fuse_include}")
    return (None, None, flags)
```

(at the class definition level) then packages that inherit `Autotools` (like `charliecloud`) will automatically set arguments like this.

Without something like this, there can be conflicts between `compilers.yaml` flags and those injected by the package.

Reply:
Done, and added fuse3 requirement for the squashfuse dep because charliecloud@0.26: only uses -lfuse3 flag...